set.mm

```diff
@@ -31823,10 +31802,9 @@ requires either a dv condition ( ~ sb56 ) or a non-freeness hypothesis
 
   ${
     $d w x $.  $d w y $.
-    $( Alternate version of ~ axc11nlemOLD2 used in an older proof.
-       (Contributed by NM, 8-Jul-2016.)  (Proof shortened by Wolf Lammen,
-       17-Feb-2018.)  (Proof modification is discouraged.)
-       (New usage is discouraged.) $)
+    $( Used in an older proof.  (Contributed by NM, 8-Jul-2016.)  (Proof
+       shortened by Wolf Lammen, 17-Feb-2018.)
+       (Proof modification is discouraged.)  (New usage is discouraged.) $)
```

Comment:
So you've removed `~aevlemALTOLD` which was the only proof using `~axc11nlemALT`. 
Do we even need to keep it?

Reply:
See discouraged file:
New usage of "axc11nlemALT" is discouraged (1 uses).

Reply:
Ok, so one usage remains, thanks!